Server/controllers/user.controller.js

```diff
@@ -81,4 +81,48 @@ exports.searchposts = (req, res, next) => {
     let results = userService.getSearchResults(req.userId, req.body);
     res.json({ searchResult: result });
 
-}
\ No newline at end of file
+}
+
+exports.followuser=(req, res, next) => {
+    console.log(req.params.uid);
+    userService.followUser(req.params.uid);
```

Comment:
followUser implementation in the userService takes in two arguments async function followUser(id,uId) but you have called it with one argument in the userController at this line. Can you please correct it?